e2e/tests/components/logical-expression-builder/logical-expression-builder.e2e-spec.ts

```diff
@@ -0,0 +1,183 @@
+import { LogicalExpressionBuilderPage } from './logical-expression-builder.po.spec';
+import { ExpressionCondition, ExpressionGroup } from '../../../../src/components/logical-expression-builder';
+
+describe('Logical Expression Builder', () => {
+    let page: LogicalExpressionBuilderPage.Page;
+
+    beforeEach(async () => {
+        page = new LogicalExpressionBuilderPage.Page();
+        await page.getPage();
+    });
+
+    it('should have correct initial state', async () => {
```

Comment:
We're not doing this "correct initial state" test anymore. e2e test cases should be use cases of the component. The relevant initial state is asserted as a prerequisite of each test case.